Prace_domowe/Praca_domowa1/Grupa2/Piotr Marciniak/homework.ipynb

```diff
@@ -0,0 +1,1512 @@
+{
```

Comment:
<p>Bardzo dobrze że to sprawdziłeś.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>